src/cli/direct.rs

```diff
@@ -1,42 +1,87 @@
-use clap::{Arg, Command};
-
-use crate::Configuration;
+use clap::{Arg, ArgAction, Command};
 
 pub fn run() -> Command {
-    Command::new("git-message-ai")
-        .about("A CLI to generate your git messages!")
+    let dry_run_arg = || {
```

Comment:
It `||` is like an arrow function but the correct name is [closures](https://doc.rust-lang.org/rust-by-example/fn/closures.html)!
I'm using it because we are using this params in more than one part so it is to avoid repeated code!